websites/serializers.py

```diff
@@ -163,6 +164,6 @@ def create(self, validated_data):
 
     class Meta:
         model = WebsiteContent
-        fields = ["type", "title", "markdown", "uuid"]
         # we want uuid in the returned result but we also want to ignore any attempt to set it on create
         read_only_fields = ["uuid"]
+        fields = read_only_fields + ["type", "title", "markdown", "metadata"]
```

Comment:
Are there separate`WebsiteContentCreateSerializer` and `WebsiteContentDetailSerializer` serializers just to make sure that once a `WebsiteContent` object is created, it's type can't be changed?  Maybe could use the same serializer but [adjust its `update` method to ignore `type` if changed](https://github.com/mitodl/ocw-studio/blob/master/websites/serializers.py#L66-L67)?  Either way works though.

Reply:
Yes, that's the intent. Adjusting the update method works but I'd rather not make the change since it would be extra work at this point